ee/clickhouse/queries/funnels/funnel_persons.py

```diff
@@ -4,6 +4,13 @@
 
 
 class ClickhouseFunnelPersons(ClickhouseFunnel):
+    def run(self, *args, **kwargs):
+        if len(self._filter.entities) == 0:
+            return []
+
+        results = self._exec_query()
+        return self._format_results(results), len(results) > self._filter.limit - 1
```

Comment:
Actually, since we're overriding `_format_results` already, (and this _seems_ like something that's to do with what format_results does), maybe this is better achieved by adding the second element to `format_results` instead?: We already have `results` and `self._filter.limit` there too.